run_dir/design/base.html

```diff
@@ -51,6 +51,7 @@
                         <li><a href="/reads_vs_qv">Reads vs QV</a></li>
                         <li><a href="/production/cronjobs">Cronjobs List</a></li>
                         <li><a href="/quotas">Quotas</a></li>
+                        <li><a href="/clusters_per_lane">Clusters per lane</a></li>
                         <li class="depreciated_page divider"></li>
                         <li id="depreciated_header"><a href="#">Depreciated Pages</a></li>
```

Comment:
Man I just thought it was a Britishism. In the future just go with that and everyone will just think I'm being an American jerk for arguing about it!